### DISCUSSION TITLE
Hide legend in statistics-graph

### DISCUSSION THREAD

**Discussion by erik3rik** [home-assistant/frontend#9846] [Dashboard]
Currently there is no way to edit the legend in the statistics-graph card.
It is very beautiful to have MIN/MAX/AVG values in the graph; but those really clutter down the graphs as each individual "thing" gets 3 spots in the legend.
![Screenshot 2021-08-19 at 16 53 51](https://user-images.githubusercontent.com/46067475/130091019-83ea961c-0a38-4e91-8a42-0e4a54a37cd0.png)

In the case that can be seen on the screenshot provided; "Outdoor Temperature" should be in the legend, not all three of them (yet I want to show them in the graph!).

I think we should have an option to select either "all" = as per the screenshot, "one" = only one per entity and "none" = no legend at all.

**Comment by Nicomagno** [comment]
This is still a thing, and worse if you have many entities in the same graph. It's a shame because the graphs look beautiful.

![Captura de pantalla 2022-12-21 110411](https://user-images.githubusercontent.com/51521824/208936498-9bb59275-ac87-4717-a62f-37bf4aa27bde.png)

Maybe not enough people requested this feature. So here I am also asking for this.

The 3 options suggested above seem like a perfect solution, maybe only one label per entity should even be the default.

**Reply by ildar170975** [reply]
What a usecase for displaying several graphs with all these "min/max/mean"?
I can understand several graphs with just "mean" line - but not with all these "min/max".

**Reply by MrPeteH** [reply]
@ildar170975 Min/Max gives a far more honest graph. Any time we show only a single data line, we are ignoring the range of the data, and thus are fooling ourselves!

**Comment by jnxxx** [comment]
> _I think we should have an option to select either "all" = as per the screenshot, "one" = only one per entity and "none" = no legend at all._

+1 for this.

**Comment by karwosts** [comment]
Starting in 2024.3, statistics legend is reworked to only have one label per entity, and the text in the parenthesis will be removed, so should be a cleaner look going forward.
